nengo/spa/tests/test_basalganglia.py

```diff
@@ -22,6 +22,7 @@ def test_basal_ganglia(Simulator, seed, plt):
             'dot(vision,CAT)+0.5-dot(vision,CAT) --> motor=E',
             'dot(vision,PARROT) + compare --> motor=F',
             '0.5*dot(vision,MOUSE) + 0.5*compare --> motor=G',
+            '( dot(vision,MOUSE) - compare) * 0.5 --> motor=H'
```

Comment:
> Why are there no spaces after commas and around operators?

That's mostly my fault, as all the examples I wrote started off that way.  I've switched over to "spaces after commas, but optional spaces around operators as per readability" which is my current understanding of PEP8.